src/main/java/org/dsa/iot/calendar/google/GoogleCalendar.java

```diff
@@ -47,8 +47,8 @@ public GoogleCalendar(Node calendarNode, String clientId, String clientSecret) t
         super(calendarNode.getChild("events"));
         this.clientId = clientId;
         this.clientSecret = clientSecret;
-        this.httpTransport = new NetHttpTransport();
-        this.jsonGenerator = JacksonFactory.getDefaultInstance();
+        httpTransport = new NetHttpTransport();
+        jsonGenerator = JacksonFactory.getDefaultInstance();
```

Comment:
It'd be great if we used dependency injection where possible. Have you ever used Dagger or Guice?